knotx-junit/src/main/java/io/knotx/junit/rule/TestVertxDeployer.java

```diff
@@ -60,9 +61,9 @@ public void evaluate() throws Throwable {
               }
               latch.countDown();
             });
-        latch.await(3, TimeUnit.SECONDS);
+        boolean await = latch.await(3, TimeUnit.SECONDS);
 
-        if (latch.getCount() != 0) {
+        if (!await || latch.getCount() != 0) {
```

Comment:
Isn't the second half of this condition superfluous?

In line 64:

    boolean await = latch.await(3, TimeUnit.SECONDS);

The `await` variable will have the value of `true` if the count has reached 0, `false` if the time elapsed but the count is not 0. Otherwise (if the thread is interrupted), it will throw an exception and the condition in the next line will not be reached.

Either way, I don't see how `(!await || latch.getCount() != 0)` could yield a different result than `!await`